react/components/icons/Delete.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+export default class Delete extends React.Component {
+
+  render() {
+    return ( 
+      <svg width="16" height="16" viewBox="0 0 16 16" fill="none">
+        <circle cx="8" cy="8" r="8" fill="#CACBCC"/>
+        <path d="M11 5L5 11" stroke="white" stroke-width="1.6" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
Does react throw on these hyphen-case props? I think it only uses camel case.

Reply:
i just copied the svg nat gave me and put it in there, i don't know but i will search

Reply:
anyway, i just convert the dash-cases to camelCases